openmc/plotter.py

```diff
@@ -47,14 +58,20 @@
 PLOT_TYPES_LINEAR = {'nu-fission / fission', 'nu-scatter / scatter',
                      'nu-fission / absorption', 'fission / absorption'}
 
+# Minimum and maximum energies for plotting (units of eV)
+_MIN_E = 1.E-5
+_MAX_E = 20.E6
+
 
 def plot_xs(this, types, divisor_types=None, temperature=294., axis=None,
-            sab_name=None, cross_sections=None, enrichment=None, **kwargs):
-    """Creates a figure of continuous-energy cross sections for this item
+            sab_name=None, ce_cross_sections=None, mg_cross_sections=None,
+            enrichment=None, plot_CE=True, orders=None, divisor_orders=None,
+            **kwargs):
+    """Creates a figure of continuous-energy cross sections for this item.
 
     Parameters
     ----------
-    this : openmc.Element, openmc.Nuclide, or openmc.Material
+    this : {openmc.Element, openmc.Nuclide, openmc.Material}
```

Comment:
Braces should only be used to indicate a set of values (usually strings). In the case of different classes that are acceptable, use the 'or' form that was there before.